curriculum/challenges/english/25-front-end-development/quiz-basic-html/66df3b712c41c499e9d31e5b.md

```diff
@@ -1373,3 +1373,464 @@ Which of the following elements should NOT be placed inside of the `head`?
+/>
+```
+
+### --question--
+
+#### --text--
+
+What does `<meta charset="utf-8" />` do?
+
+#### --distractors--
+
+Sets the language of the webpage to English.
+
+---
+
+Links an external CSS stylesheet.
+
+---
+
+Specifies the title of the webpage.
+
+#### --answer--
+
+Defines the character encoding to ensure proper text display.
+
+### --question--
+
+#### --text--
+
+What is the difference between `link` and `script` elements in HTML?
+
+#### --distractors--
+
+`link` is for JavaScript files, and `script` is for CSS files.
+
+---
+
+Both `link` and `script` are used to style web pages.
+
+---
+
+`script` is for adding styles, and `link` is for running JavaScript.
+
+#### --answer--
+
+`link` is used to add styles from an external CSS file, while `script` is used to run JavaScript code.
+
+### --question--
+
+#### --text--
+
+Which of the following statements about HTML classes and IDs is incorrect?
+
+#### --distractors--
+
+A class name can be used multiple times throughout an HTML document.
+
+---
+
+The `rel` attribute defines the relationship between the linked resource and the HTML document.
+
+---
+
+The `og:image` property is used to specify the image displayed in social media posts.
+
+#### --answer--
+
+ID names cannot contain underscores and must be unique.
+
+### --question--
+
+#### --text--
+
+Which of the following statements about HTML heading elements is true?
+
+#### --distractors--
+
+The text size increases from `h1` to `h6`.
+
+---
+
+The text size remains the same for all heading elements.
+
+---
+
+`h3` is larger than `h2`.
+
+#### --answer--
+
+The text size decreases from `h1` to `h6`.
+
+### --question--
+
+#### --text--
+
+Which statement is incorrect about HTML elements?
+
+#### --distractors--
+
+HTML elements are the basic building blocks of a webpage.
+
+---
+
+Some codebases include a forward slash (/) inside void elements.
+
+---
+
+A boolean attribute is one that works simply by being present or absent in a tag.
+
+#### --answer--
+
+Void elements contain content and only have a start tag.
+
+### --question--
+
+#### --text--
+
+Which of the following statements about void elements in HTML is true?
+
+#### --distractors--
+
+A void element must always include a forward slash / before closing.
+
+---
+
+A void element should never include a forward slash /.
+
+---
+
+The forward slash / is mandatory for self-closing void elements in all versions of HTML.
+
+#### --answer--
+
+Both `<img>` and `<img/>` are acceptable in modern HTML.
+
+### --question--
+
+#### --text--
+
+What is the primary purpose of comments in programming?
+
+#### --distractors--
+
+To make the code run faster.
+
+---
+
+To change how the browser displays content.
+
+---
+
+To store temporary data within the code.
+
+#### --answer--
+
+To leave notes for yourself and other developers in the code.
+
+### --question--
+
+#### --text--
+
+Which statement about HTML heading elements is correct?
```

Comment:
You already added this question earlier 

```md
Which of the following statements about HTML heading elements is true?
```

This second question about heading elements seems like another duplicate. It would be better to just come up with a different question and set of answers here